src/sources.js

```diff
@@ -15,28 +15,111 @@ export const GET_USERS = gql`
   } 
 `;
 
+export const GET_COUNTRIES = gql`
+    query Countries{
+        counties{
+            id
+            name
+        }
+    }
+`;
+
+export const UPDATE_CONTACTINFO = gql`
+
+  mutation UpdateContactInfo(
+        $id: Int!, 
+        $email: String, 
+        $phone: String, 
+        $city: String, 
+        $countryId: Int, 
+        $website: String, 
+        $avatarUrl: String, 
+        $about: String
+        ){
+      updateContactInfo(
+            id: $id, 
+            email: $email, 
+            phone: $phone, 
+            city: $city, 
+            countryId: $countryId, 
+            website: $website, 
+            avatarUrl: $avatarUrl,
+            about: $about
+            ){
+              id,
+              email
+              phone
+              city
+              country{
+                id
+                name
+              }
+              website
+              avatarUrl
+              about
+            }
+  }
+
+`;
+
 export const GET_USER = gql`
   query GetUser($id: Int!){
     user(id: $id){
-      username,
-       userRole{
-         name,
-       },
-     }
-   }
+      id
```

Comment:
All this information is needed only for the myProfile page - I would make sure when this query is being called.